### ISSUE TITLE
Bug or feature of the function "razor"

### ISSUE THREAD

**Issue by bigodmin** [OpenShot/openshot-qt#1437]
**System Details:**
* Operating System / Windows7 x64 Sp1
* OpenShot Version: 2.4.1
____________________________________________
Steps:
1.Create a project.
2. We paste the video.
3. Set the cursor in the middle of the video.
4. Press the "razor" button.
5. The function divides the video into two equal parts.
6. We put the cursor on half of the first piece of video.

Expected Result: the cursor is set to half the first piece of video and nothing else happens. To re-cut the fragment, you must press the "razor" button again.
Actual result: the first piece is cut into two equal halves.
____________________________
"It's very uncomfortable." I would like the function to cease to be active after one-time use.

**Comment by gaining** [comment]
I had the same issue on windows 10 except that I'd have to wait at least 30 secs before being able to use the razor tool after clicking it once. This problem is fixed in their daily build. Head over to their download section and choose daily builds.

**Comment by N3WWN** [comment]
The razor is a mode toggle.  While it is enabled, the clip will be cut anywhere that you hover (with the knife blade and vertical lines) and click.  No clips should be cut when you activate the razor mode.

If you wish to cut one or more clips where the playhead is located, set the playhead to the appropriate location, right click on the top of the playhead, select `Slice All` and then select the appropriate action:  `Keep Both Sides`, `Keep Left Side` or `Keep Right Side`

**Comment by bigodmin** [comment]
I meant that the razor function will be active until the user repeatedly presses the button. This is inconvenient because in the typical microsoft applications the razor function is implemented differently. Each time you use the function, you need to press the button. This is not good and not bad, but users are accustomed to this kind of algorithm of this typical function.

**Comment by DylanC** [comment]
@peanutbutterandcrackers - Razor is a mode not just a button. I think its much more useful to be able to make multiple cuts without clicking on the button again. Current behaviour is better for the workflow but I think as @ferdnyc mentioned we need to maybe rename the tooltip to make this razor mode easier to understand for the user.

**Comment by bigodmin** [comment]
To cut the entire file at once, you need to put markers and this seems to me to be longer in time.
I've looked at the piece to a certain place and immediately cut it off. Then he looked again and cut off again.

**Comment by ferdnyc** [comment]
I'm just wondering if @bigodmin is aware that OpenShot _also_ has a one-shot cutting function which cuts the current clip at the playhead position, exactly as I believe is being described: The Slice commands, which can be accessed from the clip's context menu or via the indicated keyboard bindings.

![screenshot from 2018-04-28 09-45-54](https://user-images.githubusercontent.com/538020/39397178-14d36690-4ac9-11e8-9c89-554ba2a7cd34.png)

If I perform any of the three "Slice" actions, the selected clip will be cut at the playhead position, leaving me with two clips if I chose "Keep Both...". If I used the "Keep Left..." or "Keep Right..." commands, the other side of the slice will be discarded.

So, OpenShot does have a cutting function that works as you're requesting. The Razor Tool is an alternative mode that allows the user to perform multiple cuts interactively, for situations where that would be useful instead.
